src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -164,14 +163,6 @@ export default {
 		},
 	},
 	watch: {
-		unreservedAmount(newValue, previousValue) {
```

Comment:
Why was this change needed in the experiment component but not previous CTA component?

Reply:
This watcher was there to assign correct amount for `selectedOption`. In this case was replaced by a computed prop having a better performance.